### PR TITLE
Arch machine set up improvements

### PR DESCRIPTION
**Pull request by AndreaCrotti**
Add setup.sh script to install all the needed packages.

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+WM="i3 dmenu"
+JAVA="jdk11-openjdk visualvm clojure rlwrap"
+AUDIO="pulseaudio pavucontrol"
+
+YAY_PACKAGES="teams"
+
+sudo pacman -S $NETWORK_PACKAGES $XORG $DEV_PACKAGES $LAPTOP_SPECIFIC $WEB_DEV $FONTS $WM $JAVA $AUDIO
+yay -S --noconfirm $YAY_PACKAGES
+
+# install leiningen
```
Comment (PaterJason) [inline setup.sh]: Leiningen is in the arch repos, I'd add `leiningen` to `JAVA` instead

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+LAPTOP_SPECIFIC="intel-ucode brightnessctl bluez bluez-utils blueman"
+WEB_DEV="firefox chromium lightdm lightdm-gtk-greeter"
+FONTS="ttf-liberation ttf-dejavu"
+WM="i3 dmenu"
+JAVA="jdk11-openjdk visualvm clojure rlwrap"
+AUDIO="pulseaudio pavucontrol"
+
+YAY_PACKAGES="teams"
+
+sudo pacman -S $NETWORK_PACKAGES $XORG $DEV_PACKAGES $LAPTOP_SPECIFIC $WEB_DEV $FONTS $WM $JAVA $AUDIO
```
Comment (PaterJason) [inline setup.sh]: Could add the `--needed` flag to avoid reinstalling packages

File: setup.sh (hunk 1 above)
Comment (AndreaCrotti) [reply setup.sh]: Yes thanks done

File: setup.sh (hunk 2 above)
Comment (AndreaCrotti) [reply setup.sh]: Ah nice, done

**Review by malcolmsparks — APPROVED** [review]

**Pull request merged**